sources/packages/web/src/views/institution/Login.vue

```diff
@@ -35,15 +33,20 @@
 </template>
 
 <script lang="ts">
+import { ref } from "vue";
 import { AppConfigService } from "../../services/AppConfigService";
-import { useRoute } from "vue-router";
 
 export default {
   components: {},
-
+  props: {
+    showBasicBCeIDMessage: {
+      type: Boolean,
+      required: true,
+      default: false,
```

Comment:
Do we need a default value if the property is required?

Reply:
default case is false, possibly not but for better clarification we may need this